scripts/install

```diff
@@ -6,219 +6,224 @@
 # IMPORT GENERIC HELPERS
 #=================================================
 
+source _common.sh
 source /usr/share/yunohost/helpers
 
 #=================================================
 # MANAGE SCRIPT FAILURE
 #=================================================
 
+ynh_clean_setup () {
+	ynh_clean_check_starting
+}
 # Exit if an error occurs during the execution of the script
 ynh_abort_if_errors
 
 #=================================================
 # RETRIEVE ARGUMENTS FROM THE MANIFEST
 #=================================================
 
-app=$YNH_APP_INSTANCE_NAME
 domain=$YNH_APP_ARG_DOMAIN
 path_url=$YNH_APP_ARG_PATH
 is_public=$YNH_APP_ARG_IS_PUBLIC
-admin_email=$YNH_APP_ARG_ADMIN_EMAIL
-admin_password=$YNH_APP_ARG_ADMIN_PASSWORD
-admin_locale=$YNH_APP_ARG_ADMIN_LOCALE
+admin=$YNH_APP_ARG_ADMIN
+email=$(ynh_user_get_info "$admin" mail)
```

Comment:
What is the different between `$admin` and `$email` there?

Reply:
instead of [grabbing the admin name from an email](https://github.com/YunoHost-Apps/mattermost_ynh/blob/6aa39059658650056c4bd550dee5e3a3f49cce2f/scripts/install#L213), the manifest [cf](https://github.com/YunoHost-Apps/mattermost_ynh/blob/ef36b4ad0f170d9c1395bb695630dba3be3b27e8/manifest.json#L59-L65) ask now for a Mattermost `$admin` to be selected from YunoHost users; then, the email associated [cf](https://github.com/YunoHost-Apps/mattermost_ynh/blob/ef36b4ad0f170d9c1395bb695630dba3be3b27e8/scripts/install#L30) to this user is used for `$email`